Test2/Task1/List.c

```diff
@@ -0,0 +1,77 @@
+#include "List.h"
+
+List* initList(void)
+{
+    List* list = (List*)malloc(sizeof(List));
+
+    if (list == NULL)
+    {
+        return NULL;
+    }
+    list->begin = NULL;
+    list->end = NULL;
+    list->size = 0;
+
+    return list;
```

Comment:
```suggestion
    return (List*)calloc(1, sizeof(List));
```
Это буквально то же самое, но несколько менее многословно